Test-№1.1-from-19.10.2022/task1.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#define constSizeStr 10
+
+bool isPalindrome(char *string, int length)
+{
+    bool soFarFit = true;
+    for (int i = 0; i < length / 2; ++i)
+    {
+        if (string[i] != string[length - 1 - i])
+        {
+            soFarFit = false;
+            break;
+        }
+    }
+    return soFarFit;
+}
+
+void tests(int *resultOfTest)
+{
+    char string1[constSizeStr] = "Ehehehehhe";
+    *resultOfTest = *resultOfTest * (false == isPalindrome(string1, constSizeStr));

```

Comment:
```suggestion
    *resultOfTest = *resultOfTest * !isPalindrome(string1, constSizeStr);

```
Фактически здесь написано отрицание, но излишни запутанным способом.